### PR TITLE
update bio

### PR DESCRIPTION
**Pull request by syuhei176**
change my bio.

### CODE REVIEW

File: src/components/Team/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const cards = [
     name: "Syuhei Hiya",
     img: "/img/team/syuhei.png",
     desc:
-      "Father of 2 lovely boys. Specializes in Programming Language Design. Experienced startup CEO twice and he says CEL is the best team ever."
+      "Father of 2 lovely boys. He founded the company building Backend as a Service for IoT in 2012. He transfered the ownership of the service to othe IoT company. After working at some IoT startups, he joined Cryptoeconomcis Lab in 2018."
```
Comment (daiki44) [inline src/components/Team/index.js:10]: `othe` is might be `other`.

Comment (tkmct) [reply src/components/Team/index.js:10]: `transfered` should be `transferred`.

Comment (syuhei176) [reply src/components/Team/index.js]: fixed!

File: src/components/Team/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const cards = [
     name: "Syuhei Hiya",
     img: "/img/team/syuhei.png",
     desc:
-      "Father of 2 lovely boys. Specializes in Programming Language Design. Experienced startup CEO twice and he says CEL is the best team ever."
+      "Father of 2 lovely boys. He founded the company building Backend as a Service for IoT in 2012. He transferred the ownership of the service to other IoT company. After working at some IoT startups, he joined Cryptoeconomcis Lab in 2018."
```
Comment (daiki44) [inline src/components/Team/index.js:10]: other -> another?

Comment (syuhei176) [reply src/components/Team/index.js]: fixed

**Pull request merged**